src/main/java/com/hydratereminder/HydrateReminderPlugin.java

```diff
@@ -380,17 +380,17 @@ protected String getTimeDisplay(Duration duration)
 	}
 
 	/**
-	 * <p>Handle the hydrate reset command by resetting the current hydrate interval and displaying
-	 * a reset success message in chat
+	 * <p>Handle the hydrate "hydrate" command by resetting the current hydrate interval, increasing
+	 * hydration breaks taken during the session and displaying a hydration success message in chat
 	 * </p>
-	 * @since 1.1.0
+	 * @since 2.0.0
 	 */
-	private void handleHydrateResetCommand()
+	private void handleHydrateHydrateCommand()
```

Comment:
@harikesh409 please remove this `handleHydrateHydrateCommand` method as it is causing the Codacy static code analysis to fail since it is not being used anywhere in the code.

Reply:
Sure I can remove it, I have not touched it since that was not the command that is refactored in this PR.